### DISCUSSION TITLE
Regarding `line 1:16: unexpected token: .`

### DISCUSSION THREAD

**Discussion by terryyin** [nerds-odd-e/doughnut#927] [General]
https://github.com/nerds-odd-e/doughnut/blob/46a549af1a72d8b6feb7d2ab8ec0c54c51b39fd3/backend/src/main/java/com/odde/doughnut/entities/Note.java#L116

The warning `line 1:16: unexpected token: .` is because of this line. However, it works. Cannot find any solution to remove the warning.

**Comment by yeongsheng-tan** [comment]
Will defining global escaping with the Hibernate globally_quoted_identifiers property in `persistence.xml` remove the need to use `backticks` for the tablename `notes_closure` remove the warning?
